src/sql/sqlops.proposed.d.ts

```diff
@@ -676,4 +676,11 @@ declare module 'sqlops' {
 			openEditor(position?: vscode.ViewColumn): Thenable<void>;
 		}
 	}
+
+	export interface ModelViewEditorOptions {
+		/**
+		 * Should the model view editor's context be kept around even when the editor is no longer visible?
+		 */
+		readonly retainContextWhenHidden?: boolean;
```

Comment:
Docs should say what the default is. It should probably be true, is that what it currently is?

Reply:
It is false by default

---
In reply to: [190707091](https://github.com/Microsoft/sqlopsstudio/pull/1483#discussion_r190707091) [](ancestors = 190707091)